sklearn/preprocessing/_data.py

```diff
@@ -1754,6 +1754,18 @@ def normalize(X, norm='l2', *, axis=1, copy=True, return_norm=False):
         An array of norms along given axis for X.
         When X is sparse, a NotImplementedError will be raised
         for norm 'l1' or 'l2'.
+    
+    Warning:: Risk of data leak
```

Comment:
Thanks for the PR! This is an unrelated change and shouldn't be in this PR, I think :)

Reply:
Thanks, I updated it to contain only the relevant commit.